apm-agent-plugins/apm-spring-webmvc-plugin/src/main/java/co/elastic/apm/agent/springwebmvc/SpringServiceNameInstrumentation.java

```diff
@@ -83,14 +87,10 @@ public static void afterInitPropertySources(@Advice.This WebApplicationContext a
             ServiceInfo fromServletContext = ServiceInfo.empty();
             ServletContext servletContext = applicationContext.getServletContext();
             if (servletContext != null) {
-                try {
-                    ClassLoader servletClassloader = servletContext.getClassLoader();
-                    if (servletClassloader != null) {
-                        classLoader = servletClassloader;
-                        fromServletContext = ServletServiceNameHelper.detectServiceInfo(JavaxServletApiAdapter.get(), servletContext, servletClassloader);
-                    }
-                } catch (UnsupportedOperationException e) {
-                    // silently ignored
```

Comment:
[for reviewer] the try-catch block is similar in the Servlet adapter, thus removing duplication.